pkg/cmd/cli/cmd/set/triggers.go

```diff
@@ -0,0 +1,744 @@
+package set
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"reflect"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/meta"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildutil "github.com/openshift/origin/pkg/build/util"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+	"github.com/openshift/origin/pkg/generate/app"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+	"k8s.io/kubernetes/pkg/util/sets"
+)
+
+const (
+	triggersLong = `
+Set or remove triggers for build configs and deployment configs
+
+All build configs and deployment configs may have a set of triggers that result in a new deployment
+or build being created. This command enables you to alter those triggers - making them automatic or
+manual, adding new entries, or changing existing entries.
```

Comment:
Right now manual for build configs means "removed" if the trigger can
be disabled (you have to manually do X).  It does not apply to
webhooks, since removal would lose the secret.  Deployment configs
distinguish between removed and manual.  We use automatic for
deployment configs, so the converse of that is manual.  We could use
disabled, but enabled doesn't match our terminology in use, so I'm
hesitant to say that picking a new term is better at the moment.